Foundation/NSNumber.swift

```diff
@@ -389,6 +389,13 @@ public class NSNumber : NSValue {
     override internal var _cfTypeID: CFTypeID {
         return CFNumberGetTypeID()
     }
+    
+    public override var description: String {
```

Comment:
this should probably call descriptionWithLocale: nil and that should format the number according to it's "objcType"